types/webpack/index.d.ts

```diff
@@ -947,6 +947,8 @@ declare namespace webpack {
             childrenCounters: any;
             usedChunkIds: any;
             usedModuleIds: any;
+            fileTimestamps: Map<string, number> | undefined;
```

Comment:
From these 2 seems that they cannot be undefined.

Reply:
I must've set them as `undefined` because that's what they're [technically initialized as](https://github.com/webpack/webpack/blob/master/lib/Compilation.js#L468), but you're correct: in practice, they're always defined by the time the compilation is accessed. Fixed.